support-frontend/app/controllers/Application.scala

```diff
@@ -46,10 +43,9 @@ class Application(
   amazonPayConfigProvider: AmazonPayConfigProvider,
   recaptchaConfigProvider: RecaptchaConfigProvider,
   paymentAPIService: PaymentAPIService,
-  membersDataService: MembersDataService,
+  mdapiUrl: String,
```

Comment:
Could we call this `membersDataApiUrl` just you don't have to mentally unpack the acronym whilst scanning code?

Reply:
we could and normally I'd prefer that - I only used that strange version because it is what the client side uses - making it easier to search the code.  What do you think?  (other option is to also change the client side at the same time but maybe not for this PR!)

Reply:
Yes I did notice that. I'd probably still change it but it's up to you